plugins/inventory/terraform.py

```diff
@@ -399,6 +399,7 @@ def aws_host(resource, module_name):
         # generic
         'public_ipv4': raw_attrs['public_ip'],
         'private_ipv4': raw_attrs['private_ip'],
+        'instance_state': raw_attrs['instance_state'],
```

Comment:
Changes to terraform.py should go here: https://github.com/CiscoCloud/terraform.py